src/dotnet/commands/dotnet-new/CSharp_nunittest/project.json.template

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable"
+  },
+  "dependencies": {
+    "System.Runtime.Serialization.Primitives": "4.1.1",
+    "NUnit": "3.4.1",
+    "dotnet-test-nunit": "3.4.0-beta-1"
+  },
+  "testRunner": "nunit",
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0"
+        }
+      },
+      "imports": [
+        "dotnet5.4",
+        "portable-net451+win8"
+      ]
```

Comment:
Rather than base this template off the xUnit version, why not use the version from the readme at https://github.com/nunit/dotnet-test-nunit? The only thing you need to add to that is the dependency on NUnit.

```json
{
    "version": "1.0.0-*",

    "buildOptions": {
        "debugType": "portable"
    },

    "dependencies": {
        "NUnit": "3.4.1",
        "dotnet-test-nunit": "3.4.0-beta-1"
    },

    "testRunner": "nunit",

    "frameworks": {
        "netcoreapp1.0": {
            "imports": "portable-net45+win8",
            "dependencies": {
                "Microsoft.NETCore.App": {
                    "version": "1.0.0-*",
                    "type": "platform"
                }
            }
        }
    }
}
```